services/order.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+from django.db import transaction
+
+from db.models import Order, Ticket, MovieSession
+from django.contrib.auth import get_user_model
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: Optional[str] = None
+                 ) -> Order | None:

```

Comment:
use single type of typehints
either "|" or "Optional"
+
```suggestion
def create_order(
                 tickets: list[dict],
                 username: str,
                 date: Optional[str] = None
                 ) -> Order | None:

```